### PR TITLE
Fix misspelling and misgrammar

### PR DESCRIPTION
**Pull request by chymy**
#### What type of PR is this?
/kind cleanup

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/apis/core/types.go [hunk 1]
```diff
@@ -3928,7 +3928,7 @@ type NodeSpec struct {
 	// +optional
 	ConfigSource *NodeConfigSource
 
-	// Deprecated. Not all kubelets will set this field. Remove field after 1.13.
+	// Deprecated. Not all kubelet's will set this field. Remove field after 1.13.
```
Comment (liggitt) [inline pkg/apis/core/types.go:3931]: kubelets

**Comment by oomichi** [comment]
@chymy Could you check @liggitt's comment? This seems to change unnecessary part.

**Comment by chymy** [comment]
Thank you @liggitt @oomichi , done.

**Comment by oomichi** [comment]
Thanks for updating.

/lgtm
/approve

**Pull request merged**